SwanProjects/swanprojects/utils.py

```diff
@@ -0,0 +1,107 @@
+import json
+import os
+
+
+def has_project_file(path):
+    """
+    Method to check if .swanproject exists
+    path: path to check
+    """
+    return os.path.exists(path + os.path.sep + ".swanproject")
+
+
+def get_project_info(path):
+    if has_project_file(path):
+        swanfile = path + os.path.sep + ".swanproject"
+        try:
+            with open(swanfile) as json_file:
+                data = json.load(json_file)
+            return data
+        except Exception as e:
+            print(e)
+            return {}
+    else:
+        return {}
+
+
+def get_project_path(cwd):
+    if cwd.startswith('/'):
+        cwd = cwd[1:]
+
+    paths = cwd.split(os.path.sep)
```

Comment:
There is an inconsistency in the fact that in some places you use `os.path.sep` while in some others (such as a few lines above in this very same function) you use `/`. Also there are places where you set `PATH` to `/usr/bin:..` so the implementation is very Linux-oriented anyway (which is fine). I would just not use anywhere `os.path.sep` and just stick to `/`.

Reply:
I propose you use a function to concatenate instead of doing it yourself.